cms/static/js/views/group_configuration_edit.js

```diff
@@ -1,15 +1,15 @@
 define([
-    'js/views/baseview', 'underscore', 'jquery', 'gettext',
+    'js/views/list_item_edit', 'underscore', 'jquery', 'gettext',
     'js/views/group_edit', 'js/views/utils/view_utils'
 ],
-function(BaseView, _, $, gettext, GroupEdit, ViewUtils) {
+function(ListItemEdit, _, $, gettext, GroupEdit, ViewUtils) {
```

Comment:
Add some class-level documentation about what these group_configuration classes are. They are for experiment group configurations, and are not generic "group configurations" anymore.